### PR TITLE
Update tado docs; added default for float in example

### PR DESCRIPTION
**Pull request by balk77**
## Proposed change
Example updated:
Added default for float filter, added margin in condition also other way around, changed rounding to 1 decimal

A float should have a default value, at least in sensors. It is good practice to also add it to automations.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

NA

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tado.markdown [hunk 1]
```diff
@@ -178,13 +178,13 @@ automation:
...
+    # Check if the room temp is more than 0.5 away from the tado thermostat reading
     condition:
     - condition: template
       value_template: >
-        {% set tado_temp = states('sensor.tado_temperature')|float %}
-        {% set room_temp = states('sensor.temp_sensor_room')|float %}
-        {{ (tado_temp - room_temp) > 0.5 }}
+        {% set tado_temp = states('sensor.tado_temperature')|float(20) %}
+        {% set room_temp = states('sensor.temp_sensor_room')|float(20) %}
+        {{ (tado_temp - room_temp) > 0.5 or (tado_temp - room_temp) < -0.5 }}
```
Comment (frenck) [inline source/_integrations/tado.markdown]:
```suggestion
        {{ (tado_temp - room_temp) | abs > 0.5 }}
```

File: source/_integrations/tado.markdown [hunk 2]
```diff
@@ -178,13 +178,13 @@ automation:
...
-    # Check if the room temp is more than 0.5 higher than the tado thermostat reading
+    # Check if the room temp is more than 0.5 away from the tado thermostat reading
     condition:
     - condition: template
       value_template: >
-        {% set tado_temp = states('sensor.tado_temperature')|float %}
-        {% set room_temp = states('sensor.temp_sensor_room')|float %}
-        {{ (tado_temp - room_temp) > 0.5 }}
+        {% set tado_temp = states('sensor.tado_temperature')|float(20) %}
+        {% set room_temp = states('sensor.temp_sensor_room')|float(20) %}
```
Comment (frenck) [inline source/_integrations/tado.markdown:186]: Why default to 20?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by balk77** [comment]
abs looks good!

20 is the average room temperature in a heated room, at least in Celsius
land. Zero would result in a huge offset when one sensor goes wild.

File: source/_integrations/tado.markdown (hunk 2 above)
Comment (balk77) [reply source/_integrations/tado.markdown:186]: 20 is the average room temperature in a heated room, at least in Celsius
land. Zero would result in a huge offset when one sensor goes wild.

File: source/_integrations/tado.markdown [hunk 3]
```diff
@@ -178,13 +178,13 @@ automation:
         - sensor.temp_sensor_room
         - sensor.tado_temperature
     
-    # Check if the room temp is more than 0.5 higher than the tado thermostat reading
-    condition:
+    # Check if the room temp is more than 0.5 away from the tado thermostat reading
+    condition. The sensors default to room temperature (20) when the reading is in error:
```
Comment (frenck) [inline source/_integrations/tado.markdown]: This is not a comment but YAML. This change has now broken this template.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tado.markdown [hunk 4]
```diff
@@ -178,13 +178,12 @@ automation:
         - sensor.temp_sensor_room
         - sensor.tado_temperature
     
-    # Check if the room temp is more than 0.5 higher than the tado thermostat reading
-    condition:
+    # Check if the room temp is more than 0.5 away from the tado thermostat reading condition. The sensors default to room temperature (20) when the reading is in error:
```
Comment (frenck) [inline source/_integrations/tado.markdown:181]: You've still removed `condition:` on this one, even in the last update.

Please restore that to make the template work.

**Review by frenck — APPROVED** [review]

**Pull request merged**